eng/templates/jobs/e2e-pcs-tests.yml

```diff
@@ -106,7 +104,7 @@ jobs:
     env:
       PCS_BASEURI: ${{ variables.PcsTestEndpoint }}
       PCS_TOKEN: $(GetAuthInfo.Token)
-      GITHUB_TOKEN: $(maestro-scenario-test-github-token)
```

Comment:
Should it get removed from some secret manifests? And some KeyVaults?

Reply:
If you remember its not managed in any of our secret manifest.

As for Key Vaults - I'll remote it once I remove it from release CLI as its referenced there